### DISCUSSION TITLE
Related to completion status

### DISCUSSION THREAD

**Discussion by Sahil-Shimpi** [github/india#313] [General]
I just completed watching the first video of industry sessions. I completed it without skipping, and without changing the playback speed, as per the instructions. I even downloaded the presentation and the study material. The completion status is still incomplete though. What should I do now?

**Comment by gauravjejurkar** [comment]
yesterday in session they tell us it's backend isssue they will fix it later

**Reply by Sahil-Shimpi** [reply]
Okay, thank you so much.

**Reply by Sudoo69** [reply]
@gauravjejurkar 3 of the videos have completion status on them on my account. ive watched the other 2 videos for like 2 times now[ they still show incomplete]. the thing im afraid of is if i close the tab will the 3 videos' status will still be complete??
any info or advice you can give me?

**Reply by gauravjejurkar** [reply]
also i have completed three video in self learning but it show me **in progress**  in your account it is same thing happen

**Reply by Sudoo69** [reply]
> also i have completed three video in self learning but it show me **in progress**  in your account it is same thing happen
Try using Microsoft edge it worked for me

**Comment by Benjamindutta** [comment]
I have compleated first lesson in Micrsoft learn but it is still showing 'In progress' on the FRT profile.

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
